x-pack/plugin/watcher/src/test/java/org/elasticsearch/xpack/watcher/support/WatcherIndexTemplateRegistryTests.java

```diff
@@ -189,8 +194,11 @@ public void testPolicyAlreadyExistsButDiffers() throws IOException  {
 
         Map<String, LifecyclePolicy> policyMap = new HashMap<>();
         String policyStr = "{\"phases\":{\"delete\":{\"min_age\":\"1m\",\"actions\":{\"delete\":{}}}}}";
-        LifecyclePolicy policy = registry.loadWatcherHistoryPolicy();
-        try (XContentParser parser = XContentType.JSON.xContent()
+        List<LifecyclePolicy> policies = registry.getPolicyConfigs().stream()
+            .map(policyConfig -> policyConfig.load(xContentRegistry))
+            .collect(Collectors.toList());
+        assertThat(policies, hasSize(1));
+        LifecyclePolicy policy = policies.get(0);        try (XContentParser parser = XContentType.JSON.xContent()
```

Comment:
newline here

Reply:
Not sure how that one happened, nice catch.